docs/How_To/Slack Bot/pinecone_setup.rst

```diff
@@ -0,0 +1,44 @@
+Set up Pinecone
```

Comment:
explain Pinecone is an optional component and explain benefits of using it

Reply:
@johnnykfeng can you please provide text to address this?